plugins/scaffolder-actions/scaffolder-backend-module-http-request/src/actions/run/backstageRequest.ts

```diff
@@ -85,6 +86,13 @@ export function createHttpBackstageAction(options: {
               'Option to turn request path logging off. On by default',
             type: 'boolean',
           },
+          continueOnBadResponse: {
+            title: 'Continue on error',
+            description:
+              'Return response code and body and continue to next scaffolder step if the response status is not a 2xx. False by default.',
```

Comment:
301 is also a successful response. Can I suggest the following wording.

"Return response code and body and continue to next scaffolder step if the response status shows an error. By default the step will fail if any status code is returned 400 and above."